recirq/otoc/loschmidt/tilted_square_lattice/analysis-walkthrough.ipynb

```diff
@@ -0,0 +1,373 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loschmidt Analysis Walkthrough"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "import matplotlib\n",
+    "matplotlib.rcParams.update(**{\n",
+    "    'axes.titlesize': 14,\n",
+    "    'axes.labelsize': 14,\n",
+    "    'xtick.labelsize': 12,\n",
+    "    'ytick.labelsize': 12,\n",
+    "    'legend.fontsize': 12,\n",
+    "    'legend.title_fontsize': 12,\n",
+    "    'figure.figsize': (7, 5),\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.otoc.loschmidt import TiltedSquareLatticeLoschmidtSpec\n",
+    "\n",
+    "assert TiltedSquareLatticeLoschmidtSpec, 'register deserializer'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Raw Results\n",
+    "\n",
+    "After some basic imports, we load the `ExecutableGroupResult` in its entirety using `cirq_google.workflow` tools. This is the complete raw dataset."
```

Comment:
What is this the raw dataset of?

Reply:
changed text:

## Load Raw Results

This notebook assumes you have executed the `run-simulator.py` script in this directory to execute the algorithmic benchmark on a simulator, and that the results were saved with run_id `"simulated-1"`. Otherwise, please modify the `run_id` parameter below.

After some standard imports, we load the `ExecutableGroupResult` in its entirety using `cirq_google.workflow` tools. This is the complete raw dataset.